org.adempiere.base/src/org/compiere/model/GridField.java

```diff
@@ -83,7 +83,15 @@ public class GridField
 	/**
 	 * 
 	 */
-	private static final long serialVersionUID = 496387784464611123L;
+	private static final long serialVersionUID = 405469916055906825L;
+	
+	private static final Character SPECIAL_CASE_DEFAULT = '1';
+	private static final Character SQL_DEFAULT = '2';
+	private static final Character DEFAULT_LOGIC = '3';
+	private static final Character USER_PREFERENCE_DEFAULT = '4';
+	private static final Character SYSTEM_PREFERENCE_DEFAULT = '5';
+	private static final Character PANEL_PREFERENCE_DEFAULT = '6';
+	private static final Character DATA_TYPE_DEFAULT = '7';

```

Comment:
probably should create constant for orderGetDefault in getDefault and defaultSeq in getDefaultForPanel too.

Reply:
Added in the new commit